### PR TITLE
526 no payment notification mg

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
Adds some help text when we don't find any payment information for a veteran
- No change to UI when payment info found
- No change to UI when there's an error retrieving the payment info

## Testing done
- Unit tests
- Tested locally

## Screenshots
Payment Info (no change):
screen shot 2018-10-22 at 8 57 17 pm

-----

No Payment Info:
screen shot 2018-10-22 at 9 25 51 pm

## Acceptance criteria
- [x] Implement notice to veterans if they don't have any payment info in our system.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/components/PaymentView.jsx [hunk 1]
```diff
@@ -33,23 +30,66 @@ const PaymentView = response => {
     );
   };
 
-  if (accountType !== NOBANK) {
-    accountNumberString = <p>Account number: {mask(accountNumber, 4)}</p>;
-    routingNumberString = <p>Bank routing number: {mask(routingNumber, 4)}</p>;
-    bankNameString = (
-      <p>Bank name: {bankName || srSubstitute('', 'is blank')}</p>
+  if (!accountType || !accountNumber || !routingNumber) {
+    // should we also check bankName?
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/components/PaymentView.jsx]: :grimacing: :man_shrugging:

File: src/applications/disability-benefits/526EZ/components/PaymentView.jsx [hunk 2]
```diff
@@ -33,23 +30,66 @@ const PaymentView = response => {
...
+            <li>
+              Fill out and send us a Direct Deposit Enrollment form (VA Form
+              24-0296).{' '}
+              <a
+                href="https://www.vba.va.gov/pubs/forms/vba-24-0296-are.pdf"
+                target="_blank"
+              >
+                Download VA Form 24-0296
+              </a>
+              , <strong>or</strong>
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/components/PaymentView.jsx]: Eh, Prettier. I'm developing a love / hate relationship with this.

File: src/applications/disability-benefits/526EZ/components/PaymentView.jsx [hunk 3]
```diff
@@ -33,23 +30,66 @@ const PaymentView = response => {
...
     <div>
       <PaymentDescription />
       <div className="blue-bar-block">
         <p>
           <strong>{accountTitleLabels[accountType.toUpperCase()]}</strong>
         </p>
-        {accountNumberString}
-        {routingNumberString}
-        {bankNameString}
+        <p>Account number: {mask(accountNumber, 4)}</p>
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/components/PaymentView.jsx]: :+1:

**Review by cvalarida — APPROVED** [review]

**Review by peggygannon — CHANGES_REQUESTED** [review]
@markgreenburg Hi, Mark.. A couple of tweaks from me: 
1. We had changed the header to File for increased disability compensation in an earlier PR.. I want to be sure that that was updated? The header here still says "Apply" 

2. For the no payment screen, i think we should delete the intro "This is the bank information we have on file for you...." as it's confusing to have intro and then the message below that there is no info. 
What do you think? 

3. Please tweak this sentence "You can still complete this form, but we'll need your payment information in order to pay you compensation" TO: 
You can still complete this form, but we'll need your payment information before we can pay you any compensation.

File: src/applications/disability-benefits/526EZ/components/PaymentView.jsx (hunk 2 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/components/PaymentView.jsx]: Yeah. To be fair, at least I didn't have to think about it - I just wrote one big long line and got it all fixed up on save.

**Comment by markgreenburg** [comment]
Thanks @peggygannon ! I think we only changed the name on the all claims form? Either way, I updated the Increase form name to use 'File.' I also tweaked the content for the 'no payment' content per your suggestions. Here's a new screenshot:

screen shot 2018-10-24 at 10 37 17 am

Please take another look when you get a chance.

**Review by peggygannon — APPROVED** [review]

**Pull request merged**